static/css/v3/heros.css

```
@@ -486,6 +487,57 @@
   padding-top: var(--space-hero-padding-top);
 }
 
+/* Positioning context for the alert below. */
+.hero:has(> .banner) {
+  /* Not var(--header-height): outside .header that resolves to an unrelated
+     legacy 2.5rem, which would tuck the alert under the header. 48px is
+     .header's own value; the rest of its footprint is its vertical padding. */
+  --hero-header-footprint: calc(48px + 2 * var(--space-default));
+  --hero-banner-inset: var(--space-large);
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)